ApiClient/Source/Pages/Queries/HomePageQuery.swift

```diff
@@ -26,26 +26,42 @@
 import Foundation
 import Alamofire
 
-final class HomePageQuery: Query {
-    typealias Result = JSONPage?
-    
-    let resource: String
+final class HomePageQuery: Query {    
+    typealias Result = JSONPage
+
+    let url: URL
+    let host: String = "https://api.schedjoules.com/pages"
     let method: HTTPMethod = .get
     let parameters: Parameters = [:]
     let headers: HTTPHeaders = ["Accept" : "application/json", "Content-Type" : "application/json"]
 
-    required init(resource: String) {
-        self.resource = resource
+    required init?(path: String?, queryItems: [URLQueryItem]) {
```

Comment:
why do you take a path if all convenience ctors pass `null` anyway and if the path is actually fixed?